--- /home/benjaminm/code/jedi-bundle/vind/src/vind/FieldsIO/FieldsIODefault.h.tmp.bak	2025-05-20 11:20:17.036440605 +0200
+++ /home/benjaminm/code/jedi-bundle/vind/src/vind/FieldsIO/FieldsIODefault.h	2025-05-20 11:23:23.213603227 +0200
@@ -12,9 +12,8 @@
 
 #include "atlas/field.h"
 
-#include "oops/base/Variables.h"
-
 #include "vind/FieldsIO/FieldsIOBase.h"
+#include "vind/VariablesSwitch.h"
 
 namespace vind {
   class Geometry;
@@ -33,7 +32,7 @@
 
   // Read
   void read(const Geometry &,
-            const oops::Variables &,
+            const varns::Variables &,
             const eckit::Configuration &,
             atlas::FieldSet &) const override;
 
